docs/content/Deployment/Production-Checklist.md

```diff
@@ -83,71 +114,7 @@ If you want to run Cube.js in production without Redis, you can use
 | to manage the query queue.
 <!-- prettier-ignore-end -->
 
-## Set up Pre-aggregations Storage
```

Comment:
We should probably do a quick sweep to make sure the headings removed below don't have any links pointing to them

Reply:
Just did a sweep, doesn't seem like anything is broken (in the docs, at least)